### ISSUE TITLE
Deprecation from ember-modifier

### ISSUE THREAD

**Issue by claire-lovisa** [appuniversum/ember-appuniversum#223]
In OP I've started to see some deprecations warnings like shown below that seem to get triggered by the `AuDropdown` component :
```DEPRECATION: ember-modifier (for onClickInside ... ```

After a little check in the app on `ember-modifier`, it seems ember-appuniversums still uses v2.1.2. Maybe a bump would fix the warning ?

```
├─┬ @appuniversum/ember-appuniversum@1.0.3
│ ├─┬ ember-inputmask5@3.2.0
│ │ └── ember-modifier@3.2.1 deduped
│ ├── ember-modifier@2.1.2
│ └─┬ ember-prop-modifier@1.0.1
│   └── ember-modifier@2.1.2
```

Thanks ! :)

**Comment by Windvis** [comment]
I think it's more complicated than simply bumping the version. It seems the latest release of ember-modifier triggers custom deprecations which need actual changes: https://github.com/ember-modifier/ember-modifier/releases/tag/v3.2.0

All our modifier subdependencies need to be updated and our own `on-click-inside` modifier also needs that change. I'll try to look into it ASAP.

**Comment by Windvis** [comment]
These are the addons that need to be updated:
- https://github.com/zestia/ember-auto-focus
- https://github.com/pzuraq/ember-prop-modifier
- https://github.com/sinankeskin/ember-inputmask

**Comment by Windvis** [comment]
The most common warning should be fixed by #234 which is part of the [1.0.6 release](https://github.com/appuniversum/ember-appuniversum/blob/master/CHANGELOG.md#v106-2022-04-15).

**Comment by Windvis** [comment]
We should probably drop both ember-prop-modifier and ember-inputmask and replace them with custom modifiers for our use-case.

ember-prop-modifier isn't maintained and ember-inputmask doesn't really work all that great if you want to do more complex things. They're also thinking of droping the lazy-loading again and we shouldn't ship inputmask as part of the main app bundle, especially if apps don't use the masking functionality.
